new-app/src/App/runGame/OTPInputGuess.tsx

```diff
@@ -55,15 +56,15 @@ const OTPInputGuess = ({guess, setGuess}: {guess: any[], setGuess: React.Dispatc
   return (
     <div>
       <div className="flex items-center justify-center">
-        <div className="m-4 p-8 border w-160 h-40 bg-gray-300 shadow-inner">
+        <div className="m-4 p-8 border-[#ff7711] border-2 w-160 h-40 bg-[#06373a] rounded-2xl shadow-inner">
```

Comment:
should we define some custom colors here?

https://tailwindcss.com/docs/customizing-colors